added multi-ISMI and Global IoT Network and Platform (#23)
EMnify/Documentation: README.adoc

=== IoT SIM

EMnify IoT SIMs are more durable than regular SIM cards and come in different <<Form Factors, form factors>> and <<Quality Grades, quality grades>>. For testing the platform services quickly without any SIM hardware - EMnify offers an <<eSIM>> which can be downloaded directly onto a supported smartphone. 
EMnify has also a <<Multi-IMSI Application, multi-IMSI software application>> directly on the SIM so different operator profiles can be used based on the location of the device. Using this technology EMnify can provide a superset of roaming networks from traditional operators. 

==== Form Factors 

The traditional, plugable SIM card comes in 4 different form factors:

> * 1FF (approximaltely size of a creditcard - only used in first GSM phone) - 85 x 54 x 0,76mm 

Reviewer: ```suggestion
* 1FF (approximately the size of a credit card - only used in first GSM phones) - 85 x 54 x 0,76mm 
```

* Cloud Connect and Datastreamer integration
* Zapier- No-Code Integrations

==== Multi-IMSI Application

> EMnify IoT SIM cards are equipped with a multi-IMSI applet that contains EMnify’s own and partner operator profiles. The different operator profiles are identified by the utilized <<IMSI>>. Each IMSI / partner operator has usually more than one network accessible per country. 

Reviewer: ```suggestion
EMnify IoT SIM cards are equipped with a multi-IMSI applet that contains EMnify’s own and partner operator profiles. The different operator profiles are identified by the utilized <<IMSI>>. Each IMSI / partner operator usually has more than one network accessible per country. 
```

> The SIM applet utilizes a preferred IMSI list per country. When a device moves to a different country which has a different preferred IMSI configured, because it for example gives access to more networks then the applet dynamically overwrites the previously active IMSI with the preferred IMSI for this country. Likewise, when an operator’s service experiences outage, the SIM can automatically fall back to a fallback IMSI to ensure connection remains uninterrupted. 

Reviewer: ```suggestion
The SIM applet utilizes a preferred IMSI list per country. When a device moves to a different country which has a different preferred IMSI configured (for e.g. because it gives access to more networks), then the applet dynamically overwrites the previously active IMSI with the preferred IMSI for this country. Likewise, when an operator’s service experiences outage, the SIM can automatically fall back to a fallback IMSI to ensure connection remains uninterrupted. 
```

The selection of the IMSI partner that is used for the countries is based on mutliple factors. The preferred IMSI selected based on: 

* allowance for permanent roaming in the country
* the most network partners in the country
* the best availability of radio access types (e.g. LTE) or availability of features (PSM/eDRX)

> The Multi-IMSI applet is transparent for the device and has no impact on the device operation. In order to analyze which IMSI is currently in use, you can either check in the EMnify portal -> Connected devices -> Details or also query via directly on the device with the AT-command `AT+CIMI?`. 

Reviewer: ```suggestion
The Multi-IMSI applet is transparent for the device and has no impact on the device operation. In order to analyze which IMSI is currently in use, you can either check in the EMnify portal -> Connected devices -> Details or also query the device directly using the AT-command `AT+CIMI?`. 
```

=== Global IoT Network and Platform 

> Even when IoT devices are often only deployed at one location and not moving - for a vendor selling into multiple countries it is important to have a global connectivity solution, so that there is not a need to have different SIM cards in stock or have multiple contracts and tariffs. For mobile use cases there is no other alternative than using n international SIM card.

Reviewer: ```suggestion
Even when IoT devices are more often only deployed at a single location and are not moving, for a vendor selling to multiple countries it is important to have a global connectivity solution, so that there is no need to have different SIM cards in stock or have multiple contracts and tariffs. For mobile use cases there is no other alternative than using an international SIM card.
```

> Therefore, for deploying IoT solutions globally it is important to <<Mobile Network Aggregation,aggregate multiple operators>> in the same tariff with one IoT SIM. Another aspect is that the platform and data routing is setup to support a global deployment, while adhering to local data privacy regulations. EMnify's global platform is therefore using a <<Distributed Data Transport,distributed data plane>> and link:https://portal.unifiedpatents.com/patents/patent/WO-2021069322-A1[patented mechanism] called <<Regional Breakout, regional breakout>> to address these needs. 

Reviewer: ```suggestion
Therefore, for deploying IoT solutions globally it is important to <<Mobile Network Aggregation,aggregate multiple operators>> in the same tariff with one IoT SIM. Another aspect is that the platform and data routing is setup to support a global deployment, while adhering to local data privacy regulations. EMnify's global platform therefore uses a <<Distributed Data Transport,distributed data plane>> and link:https://portal.unifiedpatents.com/patents/patent/WO-2021069322-A1[patented mechanism] called <<Regional Breakout, regional breakout>> to address these needs. 
```

==== Mobile Network Aggregation

> Any mobile operator has a footprint of roaming networks in foreign countries. In case any of their subscribers travel this ensures that they can be reached.  

Reviewer: ```suggestion
Any mobile operator has a footprint of roaming networks in foreign countries. In case any of their subscribers travel, this ensures that they can be reached.  
```
